sql/npc_list.sql

```diff
@@ -6191,21 +6191,21 @@ INSERT INTO `npc_list` VALUES (17047891,'_1uw','_1uw',0,140.004,-8.483,319.995,1
--- NC: INSERT INTO `npc_list` VALUES (17047904,'NOT_CAPTURED','_ju9',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047905,'NOT_CAPTURED','_jua',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047906,'NOT_CAPTURED','_jub',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047907,'NOT_CAPTURED','_juc',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047908,'NOT_CAPTURED','_jud',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047909,'NOT_CAPTURED','_jue',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047910,'NOT_CAPTURED','_juf',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17047911,'NOT_CAPTURED','_jug',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17047903,'_ju8','_ju8',0,0.000,0.000,0.000,1,50,50,0,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17047904,'_ju9','_ju9',0,0.000,0.000,0.000,1,50,50,0,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17047905,'_jua','_jua',0,0.000,0.000,0.000,1,50,50,0,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17047906,'_jub','_jub',0,0.000,0.000,0.000,1,50,50,0,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17047907,'_juc','_juc',0,0.000,0.000,0.000,1,50,50,0,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17047908,'_jud','_jud',0,100.000,0.795,-320.000,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'TOAU',0);
+INSERT INTO `npc_list` VALUES (17047909,'_jue','_jue',0,-20.000,0.772,-320.000,1,50,50,8,0,0,0,6147,0x0000320000000000000000000000000000000000,0,'TOAU',0);
+INSERT INTO `npc_list` VALUES (17047910,'_juf','_juf',0,80.000,0.796,-220.000,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'TOAU',0);
+INSERT INTO `npc_list` VALUES (17047911,'_jug','_jug',0,-20.000,-0.821,-160.000,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'TOAU',0);
 -- NC: INSERT INTO `npc_list` VALUES (17047912,'NOT_CAPTURED','_juh',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
```

Comment:
If these were captured can you link the capture data in the PR?

Reply:
https://www.dropbox.com/s/vdnnifkqbzbvqex/Breaking%20Morale%203.zip?dl=0